pydantic/fields.py

```diff
@@ -564,6 +564,13 @@ def validate(
         self, v: Any, values: Dict[str, Any], *, loc: 'LocStr', cls: Optional['ModelOrDc'] = None
     ) -> 'ValidateReturn':
 
+        if self.type_.__class__ is ForwardRef:
+            assert cls is not None
```

Comment:
In fact we could probably make `cls` required for `validate()`. AFAIK the only case where `cls` is not set is in two tests. But whatever we do I think it's best to keep with the smallest change as possible.